public/styles.css

```diff
@@ -133,7 +135,9 @@ body {
 }
 
 .overview-widget-productInfo {
+  font-size: large;
```

Comment:
don't use broad style declarations like this. Always declare in relative units such as rem/em